proposals/NNNN-Dynamic-App-List-Sorting.md

```diff
@@ -74,12 +74,12 @@ Currently, there is no sequence provided that demonstrates how the icon is displ
 
 ## Potential downsides
 
-There is no potential downside since it is a modification to the current process of the SDL app icon display order.
+Since it is necessary to implement the large amount of modification due to implement multiple sorting methods, the developer must do the implementation carefully and also, it will take a long time.
 
 ## Impact on existing code
 
-The existing sorting process is left as it is. Furthermore, as processes/lists related to sorting are added, SDL Core is affected. Also, since the priority information of `apphmitype_sorting_list` is stored, the policy server is affected, too. However, I believe their changes are minor version changes because there are no changes in RPC and parameters.
+The existing sorting process is left as it is. Furthermore, as processes/lists related to sorting are added, SDL Core is affected. Also, since the priority information of `appHMIType_sorting_list` is stored, the policy server is affected, too. However, I believe their changes are minor version changes because there are no changes in RPC and parameters.
 
 ## Alternatives considered
 
-None.
+The alternative sorting method is user defined sort order. This sorting method would be a feature that some OEMs would want. However, since this specific sorting method would only be part of the HMI and have no other SDL component integration, we think it is difficult for SDL component to realize this function. Therefore, we think it is better this function to include into the HMI Integration Guidelines and SDL Core Guides documentation.
```

Comment:
>Therefore, we think it is better this function to include into the HMI Integration Guidelines and SDL Core Guides documentation.

Are you suggesting adding the alternative sorting method (user defined sort order) to the HMI Integration Guidelines and SDL Core Guides documentation in addition to your proposed solution of dynamic app list sorting? Or are you referencing the new dynamic app list sorting here, and saying that it should be documented in the HMI Integration Guidelines and SDL Core Guides?

Reply:
@theresalech -san,
Thank you for your review.
What I proposed as the alternative sorting method is to add the user defined sort order function.
So, I will remove the sentence "Therefore, we think it is better this function to include into the HMI Integration Guidelines and SDL Core Guides documentation." .
Thank you.